src/AppBundle/Resources/assets/js/components/app.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready(function () {
+
+    // animation on inputs
+
+    const $forms = $('form');
+
+    $forms.each(function() {
+        const $form = $(this);
+        const $formInputs = $form.find('.input-wrap__input');
+
+        $form.on('change', () => {
+            $formInputs.each(function() {
+                const $input = $(this);
+                $input.toggleClass('active', $input.val().length > 0);
+            });
+        });
+    });
+
+    // animation on login button if form is valid
+    // TODO : check if the form is valid
+
+    const $buttonLogin = $('[data-button-login]');
+
+    if ($buttonLogin.length > 0) {
+        const buttonPos = $buttonLogin.offset();
+        const $loader = $('<div class="loader-login"></div>');
+
+        $loader.css('top', buttonPos.top + 'px');
+
+        $buttonLogin.after($loader);
+
+        $buttonLogin.click((e) => {
+            $loader.addClass('valid');
+        });
+    }
+
+    const $elementAppear = $('[data-anim-appear]');
+
+    function appear() {
+      $elementAppear
```

Comment:
`$elementAppear` should be passed as function argument instead of depending on the function context
```js
function appear($elementAppear) {
```